wordpress-mobile/WordPress-iOS
# Issues/reader content sanitization

This PR takes the text sanitization and formatting code used by the ReaderPostServiceRemote, ports it to Swift and moves it into a separate helper utility.  Tests were updated accordingly. One thing to note is the video press formatting logic is removed with this PR.  Modern videopress is delivered as an iframe, not a collection of divs, script tags, and flash embeds and the check was obsolete. 

This sets the stage for comments to adopt the same sanitization in #6251. 

To Test:
Run the updated tests and ensure everything passes. 
Ensure that nothing looks amiss in the reader detail (e.g. no extra space between paragraphs)

Needs review: @kurzee, would you mind taking this one as its necessary for the comment refresh.
